cms.php

```diff
@@ -4,23 +4,34 @@
 $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE,
     PDO::FETCH_ASSOC);
 
-$query= $db->prepare("SELECT `email`, `phone`, `about`,`deleted`
+$query= $db->prepare("SELECT `email`, `phone`, `about`
 FROM `cms` WHERE `deleted` = 0 ORDER BY `id` DESC LIMIT 1;");
 
-$query1= $db->prepare("SELECT `project_id`, `project_img`, 
-`project_title`,`project_text`, `project_link` FROM `projects` WHERE 
-`project_delete` = 0 ORDER BY `project_id` ASC;");
+$query2= $db->prepare("UPDATE `cms` SET `deleted`= 1;");
 
+$query3= $db->prepare("INSERT INTO `cms` (`email`, `phone`, `about`)
+VALUES (:email_out, :phone_out, :about_out);");
+
+if (!empty($_POST)) {
+    $query2->execute();
+}
+
+$email_out = $_POST ['email_out'];
+$phone_out = $_POST ['phone_out'];
+$about_out = $_POST ['about_out'];
+
+$query3->bindParam(':email_out', $email_out);
```

Comment:
you could just stick `$_POST` inside the `bindParam()` to save space.

e.g. `$query3->bindParam(':email_out', $_POST['email_out'])`